app/forms/idv/doc_pii_form.rb

```diff
@@ -61,5 +67,16 @@ def name_error
     def dob_error
       I18n.t('doc_auth.errors.lexis_nexis.birth_date_checks')
     end
+
+    def dob_min_age_error
+      I18n.t('doc_auth.errors.lexis_nexis.birth_date_min_age')
+    end
+
+    def age(dob_str)
+      dob = Date.parse(dob_str)
+      now = Time.zone.today
+      now.year - dob.year -
+        ((now.month > dob.month || (now.month == dob.month && now.day >= dob.day)) ? 0 : 1)
+    end
```

Comment:
I think if we combined this with the `dob_meets_min_age?` check we could get something simpler overall:

```ruby
today = Date.today
min_age_birthday = Date.new(today.year - AppConfig.env.idv_min_age_years.to_i, today.month, today.day)
dob = Date.parse(dob_str)

dob >= min_age_birthday
```

Reply:
This will not work for a leap year

Reply:
However, I do like the idea of combining them...

Reply:
It will work for a leap year, because we're keeping the month and day intact

Reply:
Today is 2/29/2024

Reply:
oooo I see, good catch

Reply:
Looks like ActiveSupport does the right thing:

```
Date.new(2024, 2, 29) - 13.years
# => Mon, 28 Feb 2011
```